ms-authentication/routes/controller.go

```diff
@@ -0,0 +1,34 @@
+// Copyright © 2022 Intel Corporation. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+package routes
+
+import (
+	"fmt"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+)
+
+type Controller struct {
+	service interfaces.ApplicationService
+}
+
+func NewController(service interfaces.ApplicationService) Controller {
```

Comment:
Just pass the logging client, since that is the only dependency that needs to be injected.

Reply:
I think the service client is needed for the AddAllRoutes function, so b/c service has lc Jim suggested just using service

Reply:
Yes! I missed that usage.